### PR TITLE
Edited a typo

### PR DESCRIPTION
**Pull request by superbspeed**

### CODE REVIEW

File: doc/python/v4-migration.md [hunk 1]
```diff
@@ -270,7 +270,7 @@ Prior to version 4, when plotly.py was passed a `datetime` that included a timez
 
 <!-- #region -->
 ### Headless image export on Linux with Xvfb.
-In version 4, the static image export logic attempts to automatically detect whether to call the orca image export utility using Xvfb.  Xvfb is needed for orca to work in a Linux environment if an X11 display server is not available. By default, Xvfb us used if plotly.py is running on Linux if no X11 display server is detected and `Xvfb` is available on the system `PATH`.
+In version 4, the static image export logic attempts to automatically detect whether to call the orca image export utility using Xvfb.  Xvfb is needed for orca to work in a Linux environment if an X11 display server is not available. By default, Xvfb is used on Linux if no X11 display server is detected and `Xvfb` is available on the system `PATH`.
```
Comment (emmanuelle) [inline doc/python/v4-migration.md]:
```suggestion
In version 4, the static image export logic attempts to automatically detect whether to call the orca image export utility using Xvfb.  Xvfb is needed for orca to work in a Linux environment if an X11 display server is not available. By default, Xvfb is used if plotly.py is running on Linux if no X11 display server is detected and `Xvfb` is available on the system `PATH`.
```

**Comment by emmanuelle** [comment]
Thank you for spotting the typo (us/is). I would still mention "if plotly.py is..." for maximum clarity.

**Comment by jdamiba** [comment]
@superbspeed Please commit @emmanuelle's suggestion and then we can merge this! Thanks for your help!

**Comment by superbspeed** [comment]
Thanks for the suggestion @emmanuelle. :)

**Comment by jdamiba** [comment]
💃

**Pull request merged**